### DISCUSSION TITLE
Domain and DNS

### DISCUSSION THREAD

**Discussion by Ayodejioladimeji** [vercel/community#4148] [Help] [answered]
### Summary

I'm getting this error, and i only have one vercel account and also i remove my domain everywhere its been used, but the error still persists, please help out.
Find the error below : 

"Another Vercel account is using this domain. Please set the following TXT record on _vercel.ayodejioladimeji.com to use www.ayodejioladimeji.com in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed"
![snip](https://github.com/vercel/community/assets/66247222/05590970-9980-4412-9ba6-7ce42da2ea8a)

### Steps to Reproduce

Try to add my domain to my vercel project i just created

**Comment by amyegan** [comment, marked as answer]
Hey @Ayodejioladimeji. All you need to do is add that TXT record with your domain provider to [verify domain access](https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access). Then the domain will be associated with your project. 🙂

**Reply by Ayodejioladimeji** [reply]
Thank you for your response, it really helped
